tasks/get-clair-db.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-clair-db
+spec:
+  steps:
+    - name: get-db
+      image: quay.io/redhat-appstudio/clair-in-ci:latest
+      script: |
+        #!/usr/bin/env bash
+
+        #First check if database is reachable
+        http_code=$(curl -s -o /dev/null -I -w "%{http_code}" $db_url)
+
+        if [[ "$http_code" != "200" ]]
+        then
+          echo "!!ERROR!! Response code is not 200, database could not been reached.Tested database URL: https://clair-sqlite-db.s3.amazonaws.com/matcher.zst. Response code: $http_code"
+          exit 0
+        fi
+
+        microdnf install wget zstd && wget -q https://clair-sqlite-db.s3.amazonaws.com/matcher.zst && zstd -o $(workspaces.clair-ws.path)/matcher.db -d matcher.zst
```

Comment:
Turned out this 555 MB (https://clair-sqlite-db.s3.amazonaws.com/matcher.zst) file is compressed database which is being uncompressed each time the scan runs. And now for the bad news: uncompressed DB(matcher.db) is 12 GB large. So are we ok with having 12 GB sidecar image?

Reply:
The way that this could be handled is that we will have 555 MB sidecar image. Database from it will be extracted before the scan runs (this seems for me lika plausible solution).

Reply:
The issue is that on stage cluster there is limitation of ephemeral-storage to 7Gi -> https://github.com/codeready-toolchain/host-operator/blob/master/deploy/templates/nstemplatetiers/appstudio/ns_appstudio.yaml#L84